src/test_l1_tvSettings.c

```diff
@@ -332,19 +331,8 @@ void test_l1_tvSettings_positive_RegisterVideoFormatChangeCB (void)
 	callbackData.cb = tvVideoFormatChangeHandler;
```

Comment:
how about the callbackData.userdata?  what if it is NULL? Should be considered all through the file in the similar scenarios.

Reply:
As discussed, please update this. As NULL is verified in negative scenario, we should make sure that it is holding a valid info in other scenarios.

Reply:
Done